Mage.Server/src/main/java/mage/server/Main.java

```diff
@@ -358,7 +371,14 @@ public void setMBeanServer(MBeanServer server) {
 
         @Override
         public void setInvoker(ServerInvoker invoker) {
-            ((BisocketServerInvoker) invoker).setSecondaryBindPort(managerFactory.configSettings().getSecondaryBindPort());
+            if (managerFactory.configSettings().isMultiHome()) {
+                final List<Integer> secondaryBindPorts = managerFactory.configSettings().secondaryPorts();
+                ((BisocketServerInvoker) invoker).setSecondaryConnectPorts(secondaryBindPorts);
+                ((BisocketServerInvoker) invoker).setSecondaryBindPorts(secondaryBindPorts);
+            } else {
+                ((BisocketServerInvoker) invoker).setSecondaryConnectPort(managerFactory.configSettings().getSecondaryBindPort());
```

Comment:
This is the only change in the legacy configuration: in the removed line `361` only the secondary bind port is set, while now the secondary connect port is also set. This should not produce any regression and manual testing confirmed such, however, this line can easily be removed if it proves problematic.